pkg/grizzly/parsing.go

```diff
@@ -166,12 +247,24 @@ func MarshalYAML(resource Resource, filename string) error {
 	if err != nil {
 		return err
 	}
+	return writeFile(filename, []byte(y))
+}
+
+func MarshalJSON(resource Resource, filename string) error {
```

Comment:
I'd either:
- Rename it to something more specific.
- Or change the signature to just expect the `Spec`.

Otherwise, it may lead to confusions, cause within the Grizzly package there would be two very similar methods (`MarshalJSON` & `MarshalYAML`) with the same signature, that do different things (not only using either JSON or YAML as data format).